pkg/cli/info_test.go

```diff
@@ -50,6 +50,26 @@ func TestInfo(t *testing.T) {
 			wantErr: false,
 			wantOut: "./testdata/info/info_test.txt",
 		},
+		{
+			// TODO: Write a test that adequately covers -A
```

Comment:
You should be able to implement this to get test running 
https://github.com/acorn-io/acorn/blob/34eb2440a787ff78b0717bf1a479c678e75658d4/pkg/cli/testdata/MockClient.go#L614